.github/workflows/auto-update.yml

```diff
@@ -0,0 +1,44 @@
+name: Pre-commit auto-update
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+  workflow_dispatch:
+  pull_request:
+
+jobs:
+  auto-update:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Generate token
+        uses: tibdex/github-app-token@v1
+        id: generate-token
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install pre-commit
+        run: pip install pre-commit
+
+      - name: Run pre-commit autoupdate
+        run: pre-commit autoupdate
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ steps.generate-token.outputs.token }}
+          branch: update/pre-commit-autoupdate
+          branch-suffix: short-commit-hash
```

Comment:
Do we need this branch-suffix?

Reply:
I think we need to give it a unique identifier so that branch names do not conflict.

Reply:
Mm... conflict to what? :thinking:

Reply:
@kenji-miyake 
I think branches with the same name can be problematic if there are multiple updates. Therefore, this option helps to avoid errors.

Reply:
When new updates come, create-pull-request will automatically update the PR branch. So I don't think branch-suffix is necessary.

Reply:
Thanks for showing me correct behavior. In this case, we don't need to consider conflicts. Removing...

Reply:
bab3a82